### ISSUE TITLE
bug on comment line

### ISSUE THREAD

**Issue by exdeniz** [ThisIsManta/vscode-stylus-supremacy#12]
Example:
```
.uiTextInputGroupFocus
    label
        color #44c8f5
        font-size 11px
        transform translateY(-20px)
        top -20px
    input, textarea
        border-bottom-color #44c8f5
```
comment line transform translateY(-20px) and:
```
.uiTextInputGroupFocus
    label
        // transform translateY(-20px)
        top -20px
        color #44c8f5
        font-size 11px
        // transform translateY(-20px)
    // transform translateY(-20px)
    input,
    textarea
        border-bottom-color #44c8f5
```

**Comment by ThisIsManta** [comment]
Hi @exdeniz, would you mind sharing your formatting options?

**Comment by exdeniz** [comment]
```
"stylusSupremacy.insertColons": false,
  "stylusSupremacy.insertSemicolons": false,
  "stylusSupremacy.insertBraces": false,
  "stylusSupremacy.insertNewLineBetweenSelectors": true,
  "stylusSupremacy.insertLeadingZeroBeforeFraction": false,
  "stylusSupremacy.alwaysUseImport": true,
  "stylusSupremacy.alwaysUseZeroWithoutUnit": true,
  "stylusSupremacy.reduceMarginAndPaddingValues": false,
  "stylusSupremacy.sortProperties": [
    "lost-column",
    "lost-flex",
    "triangle",
    "box",
    "flex",
    "display",
    "position",
    "flex-direction",
    "justify-content",
    "vertical-align",
    "float",
    "top",
    "right",
    "bottom",
    "left",
    "sizi",
    "width",
    "height",
    "max-width",
    "min-width",
    "max-height",
    "min-height",
    "z-index",
    "opacity",
    "overflow",
    "overflow-x",
    "overflow-y",
    "visibility",
    "margin",
    "margin-top",
    "margin-right",
    "margin-bottom",
    "margin-left",
    "padding",
    "padding-top",
    "padding-right",
    "padding-bottom",
    "padding-left",
    "background",
    "background-color",
    "background-image",
    "background-size",
    "background-repeat",
    "content",
    "box-sizing",
    "pointer-events",
    "border",
    "border-top",
    "border-right",
    "border-bottom",
    "border-left",
    "border-width",
    "border-style",
    "border-color",
    "border-spacing",
    "border-collapse",
    "border-radius",
    "clip",
    "list-style",
    "color",
    "font",
    "font-family",
    "font-size",
    "font-style",
    "font-weight",
    "font-stretch",
    "line-height",
    "letter-spacing",
    "text-align",
    "text-indent",
    "text-transform",
    "text-decoration",
    "text-shadow",
    "text-overflow",
    "whitespace",
    "outline",
    "cursor",
    "box-shadow",
    "backface-visibility",
    "will-change",
    "transition",
    "transform",
    "animation"
  ],
  "stylusSupremacy.insertNewLineAroundBlocks": false,
  "stylusSupremacy.insertNewLineAroundOthers": false,
  "stylusSupremacy.insertNewLineAroundProperties": false
```

**Comment by ThisIsManta** [comment]
I will take care of this. No promises on the date though.

To work this around, you must use `/* ... */` instead of `// ...` comment syntax.

**Comment by exdeniz** [comment]
Ok, i will use ```/* ... */``` for comment, but ```// ...``` faster to use when developing

**Comment by ThisIsManta** [comment]
This has been fixed at version 1.4.2. Please update the extension.
If the problem persists, please let me know.

Thank you for reporting!

**Comment by exdeniz** [comment]
Thanks, work is perfect now.
